snowshu/core/graph.py

```diff
@@ -100,12 +100,13 @@ def build_graph(self, configs: Configuration) -> None:
                 except IndexError:
                     continue
                 cycle_names.append(cycle_name)
-            
+
             logger.error(
-            f"""The dependency graph generated by the given specified relations yields a cyclic graph.
```

Comment:
Preferable way (even though it may be not followed entirely here or in other our repos) to use lazy logging instead of f-strings (f-strings are better for ALL other cases).
Read here https://medium.com/swlh/why-it-matters-how-you-log-in-python-1a1085851205, for example:
`logging.debug("Result x = %s, y = %d", x, y)` or `logging.debug("Result x = %(x)d, y = %(y)d", {"x": x, "y": y})`
The message does not need to be formatted when the logging level is set higher than the value set for the logger